Dockerfile

```
@@ -0,0 +1,16 @@
+# ベースイメージを選択（Python 3.9を使用）
+FROM python:3.9
+
+# アプリケーションのディレクトリを作成
+WORKDIR /MioGatto
+
+# ホストのカレントディレクトリにあるファイルをコンテナの/MioGattoディレクトリにコピー
+COPY .. /MioGatto/
+
+# アプリケーションに必要なパッケージをインストール
+RUN apt-get update\
+ && apt-get install -y python3-pip\
+ && pip install -r requirements.txt
+
+# アプリケーションを実行
+ENTRYPOINT ["bash","./startup.sh"]
```

```suggestion
ENTRYPOINT ["python","-m", "server", "--host", "0.0.0.0"]
```

こんな感じで，直接書くだけで引数とかに関する挙動はそのままでいけそうです．
実行文を長くする必要があってこの方法だと困りそうなときにまた改めて startup.sh を導入すれば良いような気がします．